### DISCUSSION TITLE
Latest Rocket version provides better TLS support

### DISCUSSION THREAD

**Discussion by NeuralNelly** [dani-garcia/vaultwarden#2574] [Q&A]
In the Rocket Docs the warning regarding Rocket use in production has changed and no longer matches the text reproduced in the Vaultwarden Wiki. [New Rocket Docs warning](https://rocket.rs/v0.5-rc/guide/configuration/#tls):

> Warning: Rocket's built-in TLS supports only TLS 1.2 and 1.3. This may not be suitable for production use.

In light of this update, has the guidance on setting up TLS using Rocket vs Caddy changed at all?

**Comment by BlackDex** [comment]
Not really actually.
Mostly because using Rocket only will not work with websockets currently. If you don't want websockets, then it could work.
